bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -359,4 +367,33 @@ public String getName() {
         return FormContainer.super.getName();
     }
 
+    @JsonIgnore
+    public String getFormDefinition() {
+        String result = null;
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            Writer writer = new StringWriter();
+            // return publish view specific properties only for runtime
+            mapper.writerWithView(Views.Publish.class).writeValue(writer, this);
+            result = writer.toString();
+        } catch (IOException e) {
+            logger.error("Unable to generate json from resource");
+        }
+        return result;
+    }
+
+    @Override
+    @Nullable
+    @JsonIgnore
+    public Boolean isEdgeDeliveryRequest() {
```

Comment:
You don't need to write @JsonIgnore here, if it is already part of interface

Reply:
@deepprakash345 - Discused with @rismehta move utility and use it in HTML similar to parser code

Reply:
done